src/helpers/utils.js

```diff
@@ -47,16 +48,22 @@ exports.getPageOptions = (route, pages, locales, pagesDir) => {
   if (!pageOptions) {
     return options
   }
-  // Construct options object
-  Object.keys(pageOptions).forEach((locale) => {
-    // Remove disabled locales from page options
-    if (pageOptions[locale] === false) {
-      options.locales = options.locales.filter(l => l !== locale)
-    } else if (typeof pageOptions[locale] === 'string') {
-      // Set custom path if any
-      options.paths[locale] = pageOptions[locale]
-    }
-  })
+
+  // Remove disabled locales from page options
+  options.locales = options.locales.filter(locale => pageOptions[locale] !== false)
+
+  // Construct paths object
+  options.paths = options.locales
```

Comment:
We probably don't want to reassign `options.path` here.

```suggestion
  options.locales
```

Reply:
That was so wrong :/
I was using reduce before, forgot to change that line. Thanks!